### PR TITLE
Reddit - finished version

### PR DESCRIPTION
**Pull request by NullVoxPopuli**
TODO:
Finish

### CODE REVIEW

File: .template-lintrc.js [hunk 1]
```diff
@@ -1,5 +1,8 @@
 'use strict';
 
 module.exports = {
-  extends: 'octane'
+  extends: 'octane',
+  rules: {
+    'require-button-type': false
```
Comment (Alonski) [inline .template-lintrc.js]: Why? 🤔 😜

Comment (NullVoxPopuli) [reply .template-lintrc.js]: Why does a button need a type? :/

Comment (Alonski) [reply .template-lintrc.js]: https://github.com/ember-template-lint/ember-template-lint/blob/master/docs/rule/require-button-type.md
Most buttons aren't "submit" buttons :)

Comment (NullVoxPopuli) [reply .template-lintrc.js]: but buttons outside of a form with type button feels redundant. why is default submit. :-\

Comment (Alonski) [reply .template-lintrc.js]: Ha blame the web

**Pull request closed without merging**